src/Expensify.tsx

```diff
@@ -254,7 +241,7 @@ function Expensify(props) {
                     <NavigationRoot
                         onReady={setNavigationReady}
                         authenticated={isAuthenticated}
-                        lastVisitedPath={props.lastVisitedPath}
+                        lastVisitedPath={lastVisitedPath}
```

Comment:
```suggestion
                        lastVisitedPath={lastVisitedPath as Route}
```

This way you solve your issues with this prop. Import `Route` from `'./ROUTES'`